### ISSUE TITLE
[FEATURE] Support CLAM and TRIDENT data format

### ISSUE THREAD

**Issue by lxr2** [Franblueee/torchmil#17]
## 🎯 Problem Statement

Thank you for creating this repository! After reading through the repo documentation, I found that the current torchmil dataset only supports one Directory structure. This isn't a problem, but considering we've been using CLAM and TRIDENT to generate patch features, it would be great if these two formats could be directly supported!

Specifically, for features_path, I hope it can support both .pt and .h5 formats, corresponding to CLAM and TRIDENT data formats respectively. Additionally, CLAM and TRIDENT save coords in h5 files. Also, I hope labels can support dataframe format, which would enable direct compatibility with CLAM training code.
